torch/csrc/jit/script/compiler.cpp

```diff
@@ -721,6 +721,24 @@ std::shared_ptr<SugaredValue> BuiltinFunction::call(
       emitBuiltinCall(loc, *m.graph(), symbol, inputs, attributes, true));
 }
 
+inline bool isSupportedListElementType(TypePtr type) {
+  return type->isSubtypeOf(DynamicType::get()) ||
+      type->isSubtypeOf(NumberType::get());
```

Comment:
I think we also have lists of doubles

Reply:
It is NumberType which handles Int and Float.

Reply:
Uh of course...